src/Products/Product.java

```diff
@@ -0,0 +1,44 @@
+package Products;
+
+import Enums.KindOfProduct;
+import Enums.Models;
+import Enums.Producers;
+
+public class Product {
+
+
+    public static void main(String[] args) {
+    }
+
+    protected String tipo;
```

Comment:
nomi delle variabili e dei tipi di dato in INGLESE.

Reply:
Risolto!